typings/css-properties.d.ts

```diff
@@ -5,6 +5,10 @@ type CSSWideKeyword = 'initial' | 'inherit' | 'unset'
 
 // Based on [CSSProperties from React types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts)
 export interface CSSProperties {
+   /**
+   * Glamorous-specific property which contains any of the following CSS definitions
+   */
+  css?: CSSProperties
```

Comment:
Looking into it more, seems like ExtraGlamorousProps needs to be modified as well. I'll adjust the PR in a few.